README.md

```diff
@@ -12,12 +12,13 @@
 ```objc
 // Converts all the keys in the JSON to camelCase
 - (id)minced_JSONKeysToCamelCase;
-
-// Converts all the keys in the NSArray to camelCase
 - (NSArray *)minced_JSONObjectsKeysToCamelCase;
-
-// Converts all the keys in the NSDictionary to camelCase
 - (NSDictionary *)minced_JSONObjectKeysToCamelCase;
+
+// Converts all the keys in the JSON to camelCase and replaces null values with ""
```

Comment:
Might be better if we say

// Converts all the keys in the JSON to camelCase and replaces null values with an empty string